nixos/modules/services/web-apps/mastodon.nix

```diff
@@ -215,6 +216,25 @@ in {
         type = lib.types.str;
       };
 
+      webDomain = lib.mkOption {
+        description = ''
+          You can use this option if you want the Mastodon username domain part to
```

Comment:
The [docs](https://docs.joinmastodon.org/admin/config/#web-domain) mention that one should never change this setting for a running server. I think we should reflect that in our documentation of the option.

All in all I'm somewhat hesitant of adding this option. Wouldn't it be better to just allow people to add attributes to `env`? This way people would have to read the official docs and really know what they are doing.

Reply:
I think you are right, I will close this